starter-code/src/functions-and-arrays.js

```diff
@@ -11,13 +18,46 @@ var words = [
   'crackpot'
 ];
 
-// Calculating a Sum
+function findLongestWord(){
+var longitud = 0;
+var mPalabra = "";
+
```

Comment:
if(words.length==0){ 
      return(undefined)
  }
Esta condición falta in muchas funciones, se utiliza para devolver false si un array esta vacío